### DISCUSSION TITLE
Increase reputation rewards for turning in WSG and AB marks

### DISCUSSION THREAD

**Discussion by djobf** [chromiecraft/chromiecraft#3728] [Suggestions]
I know the rep requirement for epic items has been reduced to revered but the 75 rep per 3 marks is way too low to reach it in a sensible amount of time, especially for AB since it only really pops when it's CTA.
I've been spamming AB for 4 CTAs and I'm still halfway through honored.
This might incentivize people to queue more during the week instead of waiting for CTA now that ZG is out.

**Comment by RestoCC** [comment]
Upvoted.

As someone who prefers PvE, I have little interest touching WSG/AB even on CTA weekends. This would definitely motivate me to play these BGs more often.

Not to mention that the rep requirement is not blizzlike and the staff already made an adjustment to AV. This does not add "flavour" to the game but rather an arbitrary grind that is all too similar to what Blizzard does in recent expansions to keep you logging in each day versus systems that cause you to want to log in for enjoyment.

**Comment by balleny** [comment]
IMHO, the restrictions should be lifted completely and made blizzlike again.
